docs/cross-references.md

```diff
@@ -66,6 +66,8 @@ Cross-referencing content is accomplished with markdown link syntax (`[text](#ta
   - [Admonition](xref:spec#admonition)
 ```
 
+[^3]: To link to a page, that file must be included in the project's [table of contents](./table-of-contents.md). If it isn't, then the document will be presented as a static file.
```

Comment:
I'm going to change this to a non-numeric footnote, so that it's stable if we make future changes above!